cli/js/lib.deno_runtime.d.ts

```diff
@@ -933,6 +933,15 @@ declare namespace Deno {
      *       assert(status.state !== "granted")
      */
     revoke(d: PermissionDescriptor): Promise<PermissionStatus>;
+    /** Requests the permission.
+     *       const status = await Deno.permissions.request({ name: "env" });
+     *       if (status.state === "granted") {
+     *         home = Deno.homeDir();
```

Comment:
If you make this line console.log(Deno.homeDir()) then this snippet of code becomes copy/paste able

Reply:
OK. Makes sense. I'll update!